types/Config.js

```diff
@@ -26,6 +26,7 @@ export type Config = {
   colors: boolean,
   coverageCollector: Path,
   coverageReporters: Array<string>,
+  collectCoverageOnlyFrom: Object,
```

Comment:
Usually I prefer `{[key: string]: Path}` for this kinda stuff because `Object` doesn't do much.